tensorflow/python/autograph/utils/ag_logging.py

```diff
@@ -46,7 +46,7 @@ def set_verbosity(level, alsologtostdout=False):
   More verbose logging is useful to enable when filing bug reports or doing
   more in-depth debugging.
 
-  There are two controls that control the logging verbosity:
+  There are two parameters that control the logging verbosity:
```

Comment:
How about; "two means to control"

Reply:
lgtm

Reply:
Great, please update the PR then. Thanks!